CHANGELOG.md

```diff
@@ -3,3 +3,31 @@
 ## 0.1 (November 6, 2018)
 
 * Initial release
+
+## 0.2 (July 22, 2019)
```

Comment:
It's not version 0.2 we released. It's 1.0.2.

Perhaps also clarify the release from November to be 0.0.1.

Reply:
Thanks!